### ISSUE TITLE
[3.3.5] Core/Creature: Stationary gun "vehicle" (27714) can move for "The Chain Gun And You" (12457)

### ISSUE THREAD

**Issue by Treeston** [TrinityCore/TrinityCore#16308]
Branch 3.3.5, rev. 8a0bbc386982

[Quest: The Chain Gun And You](http://www.wowhead.com/quest=12457/the-chain-gun-and-you)
[Creature: 7th Legion Chain Gun](http://www.wowhead.com/npc=27714/7th-legion-chain-gun)

Players can move with the Chain Gun vehicle. It is supposed to be immobile.

**Comment by Killyana** [comment]
I compared this npc to 30236 that cannot be moved, than I changed the creature_templated exactly with same values, but it still moves, no idea why this happen or how to fix it.
``` sql
SELECT * FROM `creature_template` WHERE `entry` IN (27714,30236);

**Comment by Aokromes** [comment]
Check this again, i think it was fixed.

**Comment by Killyana** [comment]
No, it must be fixed here cdb16c5181b0ad02e5dd3a4afa44151fb119b62e
